site/sfguides/src/getting_started_with_native_apps/getting_started_with_native_apps.md

```diff
@@ -86,10 +87,11 @@ Duration: 3
 To create our Snowflake Native Application, we will first clone the [starter project](https://github.com/Snowflake-Labs/sfguide-getting-started-with-native-apps) by running this command:
 
 ```bash
-git clone https://github.com/Snowflake-Labs/sfguide-getting-started-with-native-apps.git
+snow app init --template git@github.com:Snowflake-Labs/sfguide-getting-started-with-native-apps.git
```

Comment:
I suggest that we keep this as a `git clone` as we do not need the template expansion done by `snow app init`. As long as the repo itself has the `snowflake.yml` in it we're good